### ISSUE TITLE
Erroneous UNASSIGNED-CoreValidation-DevLimit-MissingQueryCount

### ISSUE THREAD

**Issue by timvpGoogle** [KhronosGroup/Vulkan-ValidationLayers#2215]
After updating the validation layers within ANGLE, we are seeing the following validation layer warning now:

```
WARN: RendererVk.cpp:340 (DebugUtilsMessenger): [ UNASSIGNED-CoreValidation-DevLimit-MissingQueryCount ] Validation Warning: [ UNASSIGNED-CoreValidation-DevLimit-MissingQueryCount ] Object 0: handle = 0x55e66dc379b0, type = VK_OBJECT_TYPE_PHYSICAL_DEVICE; | MessageID = 0x17b44fbf | vkGetPhysicalDeviceQueueFamilyProperties() is called with non-NULL pQueueFamilyProperties before obtaining pQueueFamilyPropertyCount. It is recommended to first call vkGetPhysicalDeviceQueueFamilyProperties() with NULL pQueueFamilyProperties in order to obtain the maximal pQueueFamilyPropertyCount.
```
The code that the VVL is complaining about:

https://source.chromium.org/chromium/chromium/src/+/master:third_party/angle/src/libANGLE/renderer/vulkan/RendererVk.cpp;drc=9f5eb0b8fb29b35b89858c36272db16d9b7d2302;l=843
```
    uint32_t queueCount = 0;
    vkGetPhysicalDeviceQueueFamilyProperties(mPhysicalDevice, &queueCount, nullptr);

    ANGLE_VK_CHECK(displayVk, queueCount > 0, VK_ERROR_INITIALIZATION_FAILED);

    mQueueFamilyProperties.resize(queueCount);
    vkGetPhysicalDeviceQueueFamilyProperties(mPhysicalDevice, &queueCount,
                                             mQueueFamilyProperties.data());
```

The warning is being generated by the second call to `vkGetPhysicalDeviceQueueFamilyProperties()`, so it seems the VVL is forgetting about the first call now.

**Comment by ncesario-lunarg** [comment]
Fixed in #2220.
